Automattic/wp-calypso
# A4A > Feedback: Initial implementation of in-product feedback mechanism

Related to https://github.com/Automattic/automattic-for-agencies-dev/issues/1246

## Proposed Changes

* This PR does the initial implementation of the in-product feedback mechanism. 

## Why are these changes being made?


* To ask the users for in-product feedback after some events are performed. 

## Testing Instructions



* Open the A4A live link
* Go to the marketplace > Add a referral > Verify the below screen is shown when you are redirected to the Referrals page. Please note the button won't work as expected. This will be implemented in another PR.
* Append the URL with /referrals/dashboard?flags=-a4a-product-feedback > Repeat the steps without refreshing the page and verify the feedback is not shown. 

<img width="1728" alt="Screenshot 2024-12-04 at 8 21 35 AM" src="https://github.com/user-attachments/assets/3ac4d7d3-b0f5-4550-8912-7dc86d7989da">

<img width="373" alt="Screenshot 2024-12-04 at 8 22 05 AM" src="https://github.com/user-attachments/assets/ae68b1fb-9ccd-4cb6-aca7-79359c1167eb">


## Pre-merge Checklist
